R/qza_to_tse.R

```diff
@@ -33,13 +32,13 @@
 #' @export
 #' 
 #' @author
-#' Leo Lahti
-#' Noah de Gunst
+#' Leo Lahti and Noah de Gunst
```

Comment:
place this on the same line than @author tag as it is short line

Reply:
should be good now